maven-core/src/main/java/org/apache/maven/exception/DefaultExceptionHandler.java

```diff
@@ -124,7 +115,7 @@ private ExceptionSummary handle( String message, Throwable exception )
             message = getMessage( message, exception );
         }
 
-        return new ExceptionSummary( exception, message, reference, children );
+        return new ExceptionSummary( exception, message, null, children );
```

Comment:
Are we losing the exception at all?

Reply:
No, in reference we store shot exception name or links to cwiki

Reply:
Got it
Why not dropping the field?

Reply:
I think it should be possible for some Maven Exceptions to provide their own reference, so plugins and extensions can refer to their own documentation. This cleanup looks a bit too agressive to me.